plugins/builder-dockerfile/builder-build

```diff
@@ -36,7 +38,7 @@ trigger-builder-dockerfile-builder-build() {
   declare -a ARG_ARRAY
   eval "ARG_ARRAY=($DOCKER_ARGS)"
 
-  "$DOCKER_BIN" image build "${DOCKER_BUILD_LABEL_ARGS[@]}" $DOKKU_GLOBAL_BUILD_ARGS "${ARG_ARRAY[@]}" "${DOKKU_DOCKER_BUILD_OPTS[@]}" -t $IMAGE .
+  "$DOCKER_BIN" image build "${DOCKER_BUILD_LABEL_ARGS[@]}" $DOKKU_GLOBAL_BUILD_ARGS "${ARG_ARRAY[@]}" ${DOKKU_DOCKER_BUILD_OPTS} -t $IMAGE .
```

Comment:
Why did you make this particular change?

Reply:
User defines `DOKKU_DOCKER_BUILD_OPTS` variable in `dokku config:set` command as string. Also [builder-build#L32](https://github.com/dokku/dokku/blob/ab454354730f379e5b56762bfd9d6e85b9122b85/plugins/builder-dockerfile/builder-build#L32) modifies this variable as string. For consistency it is reasonable to use variable as string, not array.

Quoted variable leads to incorrect arguments for `docker build image` command. I.e.:
`dokku config:set app DOKKU_DOCKER_BUILD_OPTS='--build-arg ENV=prod'`
leads to
`docker image build <...> '--build-arg ENV=prod' <...>`

Also empty (default) value for DOKKU_DOCKER_BUILD_OPTS variable makes `docker build image` to emit error because of quotes.

Reply:
Do you know how this might handle backticks in those strings? Basically, is it possible to set the following docker option and have it do the right thing? Mind testing?

```shell
dokku docker-options:add $APP deploy '--label traefik.http.routers.node-js-app.rule=Host(`node-js-app.dokku.arketyped.net`)'
```

Reply:
Tested using proposing changes.
```dokku config:set app DOKKU_DOCKER_BUILD_OPTS='--label traefik.http.routers.node-js-app.rule=Host(`node-js-app.dokku.arketyped.net`)'```
Trace:
```
remote: + declare -a ARG_ARRAY
remote: + eval 'ARG_ARRAY=()'
remote: ++ ARG_ARRAY=()
remote: + docker image build --label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=com.dokku.app-name=app --label=com.dokku.image-stage=build --label=org.label-schema.schema-version=1.0 --label=org.label-schema.vendor=dokku --label=dokku --label 'traefik.http.routers.node-js-app.rule=Host(`node-js-app.dokku.arketyped.net`)' -t dokku/app:latest .
```

Reply:
Did the resulting container have that tag? Mind posting the output?

Reply:
Output of `docker inspect dokku/app`:
```
            "Labels": {
                "com.dokku.app-name": "app",
                "com.dokku.image-stage": "release",
                "dokku": "",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "dokku",
                "traefik.http.routers.node-js-app.rule": "Host(`node-js-app.dokku.arketyped.net`)"
            }
```

Output of `docker inspect 9c0d4cf0ea6d`:
```
            "Labels": {
                "com.dokku.app-name": "app",
                "com.dokku.container-type": "deploy",
                "com.dokku.dyno": "worker.1",
                "com.dokku.image-stage": "release",
                "com.dokku.process-type": "worker",
                "dokku": "",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "dokku",
                "traefik.http.routers.node-js-app.rule": "Host(`node-js-app.dokku.arketyped.net`)"
            }
```

Reply:
Thanks! I'll merge once tests pass :)